src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -147,6 +148,14 @@ public function load(array $configs, ContainerBuilder $container)
         $this->registerMessengerConfiguration($config, $loader);
         $this->registerElasticsearchConfiguration($container, $config, $loader);
         $this->registerDataTransformerConfiguration($container);
+
+        if ('test' === $container->getParameter('kernel.environment')) {
```

Comment:
Maybe could we find something more elegant.

Reply:
Would accessing to kernel and doing ->isDebug() more elegant ?

Reply:
debug != testing. What I have in mind is more something like that https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L207